README.md

```diff
@@ -387,6 +393,10 @@ buildctl build ... \
   --import-cache type=gha
 ```
 
+Note that layers are saved to GitHub Actions cache. This cache currently has a [size limit of 10GB](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy) that is shared accross different caches in the repo. If you exceed this limit, GitHub will save your cache but will begin evicting caches until the total size is less than 10 GB. Recycling caches too often can result in slower runtimes overall.
```

Comment:
"Note that layers are saved to GitHub Actions cache." -> "Github Actions cache backed saves both cache metadata and layers to GitHub's Cache service."

Reply:
https://github.com/moby/buildkit/pull/2489/commits/d35f032c9e51483c1808299015a04cdfaa565566